### PR TITLE
Addresses #118, fixes unit test for SVC when probabilities is False

### PR DESCRIPTION
**Pull request by xadupre**
When probability=False, onnxruntime and scikit-learn do not return the same raw score. scikit-learn returns aggregated score with a matrix [N, C] and onnxruntime returns the raw score from svm algorithm with a matrix [N, (C(C-1)/2].

### CODE REVIEW

File: tests/test_SklearnSVMConverters.py [hunk 1]
```diff
@@ -78,13 +118,70 @@ def test_convert_svmc_linear_binary(self):
             X,
             model,
             model_onnx,
-            basename="SklearnBinSVCLinearPF",
-            allow_failure=True,
+            basename="SklearnBinSVCLinearPF-NoProbOpp",
+            allow_failure="StrictVersion(onnxruntime.__version__)"
+                          " <= StrictVersion('0.5.0')"
```
Comment (vinitra) [inline tests/test_SklearnSVMConverters.py]: Will the changes be made in ONNX Runtime by the version 0.5.0 release? Perhaps < is more appropriate that <=

Comment (xadupre) [reply tests/test_SklearnSVMConverters.py]: Good call :), the PR is already in onnxruntime so < is better.

File: tests/test_SklearnSVMConverters.py [hunk 2]
```diff
@@ -78,13 +118,70 @@ def test_convert_svmc_linear_binary(self):
...
             model,
             model_onnx,
-            basename="SklearnBinSVCLinearPF",
-            allow_failure=True,
+            basename="SklearnBinSVCLinearPF-NoProbOpp",
+            allow_failure="StrictVersion(onnxruntime.__version__)"
+                          " < StrictVersion('0.5.0')"
         )
 
-    def test_convert_svmc_linear_multi(self):
```
Comment (vinitra) [inline tests/test_SklearnSVMConverters.py]: Why were these originally named "svmc" before? I noticed the same naming structure in onnxmltools' unit tests for libSVM.

Comment (xadupre) [reply tests/test_SklearnSVMConverters.py]: Probably me, I changed for something more explicit.

**Pull request merged**